src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/Statistics/BoundaryLevelUpdateRequest.cs

```diff
@@ -0,0 +1,8 @@
+namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels.Statistics;
+
+public class BoundaryLevelUpdateRequest
+{
+    public int Id { get; set; }
+
+    public string Label { get; set; }
```

Comment:
You could consider adding some Fluent validation to this request type.
I've not seen yet if you enforce a minimum label in the FE but if you do it can be enforced here too like so...

(not tested! 🙂 )

```csharp
    public class Validator : AbstractValidator<BoundaryLevelUpdateRequest>
    {
        public Validator()
        {
            RuleFor(request => request.Id)
                .GreaterThan(0);

            RuleFor(request => request.Label)
                .NotEmpty()
                .MaximumLength(5);
        }
    }
```